ios/BlurView.m

```diff
@@ -37,12 +37,13 @@ - (void)setBlurType:(NSString *)blurType
     } else {
         self.blurEffect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleDark];
```

Comment:
The issue is probably related to using a class method on a subclassed view. Have you tried using an initializer like `[[BlurAmount alloc] init]` instead of the class method?